### DISCUSSION TITLE
bigfix/11.0/inventory/Inventory/tuning/t_rebuilding_database_indexes

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#70] [Q&A]
# bigfix/11.0/inventory/Inventory/tuning/t_rebuilding_database_indexes

Rebuilding database indexes and updating the statistics is mandatory for BigFix Inventory to operate smoothly. If you experience a delay in UI response or importing data, make sure to defragment your hard disk and that statistics are up to date.

https://help.hcltechsw.com/bigfix/11.0/inventory/Inventory/tuning/t_rebuilding_database_indexes.html

**Comment by JoeBrunsTR** [comment]
How often (daily, weekly, monthly) shoud this be run?

**Comment by AnupaGiscus** [comment]
It is recommended to run on a weekly basis, and please do not perform the operation when Data Import is in progress.

Thank you!
